index.js

```diff
@@ -1,9 +1,20 @@
 const axios = require('axios')
-const { getCorrelationId } = require('@first-lego-league/ms-correlation')
-
-exports.client = axios.create({
-  transformRequest: [(data, headers) => {
-    headers['correlation-id'] = getCorrelationId()
-    return JSON.stringify(data)
-  }]
-})
+
+const { correlate } = require('./lib/correlation')
+const { makeIndependent } = require('./lib/independence')
+
+const DEFAULT_OPTIONS = {
+  independent: false
+}
+
+exports.createClient = (options = DEFAULT_OPTIONS) => {

```

Comment:
This is not a good way to handle default options. It's works only when you have one

Reply:
So what do you suggest to do instead?